exchange.go

```diff
@@ -147,6 +149,7 @@ func (re *RabbitExchangeImpl) ReceiveFrom(name, exchangeType string, durable, au
 		RoutingKey: key,
 		AutoDelete: true,
 		Exclusive:  true,
+		Prefetch:   rabbitPrefetchForSingleQueue + runtime.NumCPU()*2,
```

Comment:
Shouldn't we parametrise this and send it from outside when calling ReceiveFrom? I really wish golang had default parameters...

Reply:
I know it would be a pain to add the extra parameter everywhere :(

Reply:
Default parameters is exactly the problem here, adding them to the language does not solve the problem :) 


So the issue is we call Consume which calls ReceiveFrom which calls Receive that does take is as a parameter (with a default of 2x NumCPU. 
I won't mind adding the parameter to ImplRabbitEventHandler, to pass it to Receive.

Reply:
I see!